README.md

```diff
@@ -104,6 +104,9 @@ will make unitypackage for AvatarOptimizer 0.2.x
   // by default, beta releases are not allowed.
   // to allow all beta versions in that range, please make this true
   "includePrerelease": false,
+  // By default, if dependencies are already installed then will notify the user with a prompt.
+  // If you don't want to prompt the user if already installed, please set this to true
+  "silentIfInstalled": false,
```

Comment:
通常用途では使ってほしくないから、他のunitypackageと結合しないときにはオンにしないのを推奨すると明記したい

Reply:
```
// For advanced usage only. This will suppress the prompt to users if dependencies are already installed.
// It is strongly recommended you keep this as the default false, otherwise users may be confused why nothing happened.
// This should only enabled in certain use cases, for example if you are distributing with other assets
```
はどうですか？

Reply:
LGTM